src/core/components/ChatView.module.css

```diff
@@ -94,11 +94,13 @@
 .assistantContainer {
   margin-top: var(--spacing-sm);
   max-width: 80%;
+  width: 80%;
 }
 
 #progressStream {
   .content {
     color: var(--colors-textTertiary);
+    font-size: 1.5rem;
```

Comment:
shrink the font a bit in the stream a tiny bit (was 1.6) for better alignment and cleaner look